frontend/src/environments/environment.common.ts

```diff
@@ -1,13 +1,19 @@
+import { dotenv } from './dotenv';
+
 export interface Environment {
   production: boolean;
   useSentry: boolean;
   sentryDSN: string;
   useGoogleTagManager: boolean;
   tagManagerId: string;
   APIUrl: string;
+  functionsKey: string;
+  mapsKey: string;
 }
 
-export const commonEnvironment: Pick<Environment, 'sentryDSN' | 'tagManagerId'> = {
+export const commonEnvironment: Pick<Environment, 'sentryDSN' | 'tagManagerId' | 'functionsKey' | 'mapsKey'> = {
   sentryDSN: 'https://bc6889e92f5d4c9d941ed23760f38d37@sentry.io/1548140',
```

Comment:
Shouldn't we move also those the environment config?

Reply:
We might keep them in dotenv, but I don't think this is needed. The only danger it provides is that we will get fake errors in Sentry if somebody decides to use our sentryDSN and we will get fake/trafic events in Google Analytics dashboard. It might be inconvenient and frustrating but not harmful IMO. Also, I don't see any point for somebody doing it unless just having some fun of messing peoples dashboards :P 

Conversely, leaking the keys out provides some real danger related to money :)

Reply:
Yeah, it's non essential, just to keep things a bit more tidy. Cool, let's leave it for later maybe.